samples/sample_vpp/include/sample_vpp_config.h

```diff
@@ -125,6 +125,15 @@ typedef struct
 
 } sDenoiseParam;
 
+struct sColorFillParam: public mfxExtVPPColorFill {
```

Comment:
as for me, it is hard to read such code, several newlines required. My suggestion:
```
struct sColorFillParam: public mfxExtVPPColorFill
{
     FilterConfig mode = VPP_FILTER_DISABLED;

     sColorFillParam()
     : mfxExtVPPColorFill()
     {
         Header.BufferId = MFX_EXTBUFF_VPP_COLORFILL;
         Header.BufferSz = sizeof(mfxExtVPPColorFill);
     }
};
```

Reply:
It's ok, but it will look differently from other similar classes.